### PR TITLE
Add vATIS profiles for Aerodrome Controllers

### PR DESCRIPTION
**Pull request by RedstonePilot**
Fixes #[issue_no] - Not really an issue but can create one if required

# Summary of changes

Add vATIS profiles for ADC.

### CODE REVIEW

**Review by SamLefevre — CHANGES_REQUESTED** [review]
Heathrow shouldn't include a London City ATIS.

**Comment by RedstonePilot** [comment]
Thanks. I'll remove it in a bit. I think it was just left over from Lenny's.

File: UK/vATIS/ADC/Gatwick(EGKK).json
Comment (SamLefevre) [inline UK/vATIS/ADC/Gatwick(EGKK).json:1]: Could you also remove this file and move the existing London Gatwick .json in here. The already existing one is highly realistic and based off the real world Gatwick ATIS.

**Review by SamLefevre — CHANGES_REQUESTED** [review]

**Comment by fcooper94** [comment]
Please use the Heathrow ATIS profile I’ve linked in the Heathrow_Rated channel. It’s pinned.

Sent from my iPhone

On 18 Apr 2024, at 08:26, Ben Walker ***@***.***> wrote:

﻿

Thanks. I'll remove it in a bit. I think it was just left over from Lenny's.

File: UK/vATIS/ADC/Gatwick(EGKK).json
Comment (RedstonePilot) [reply UK/vATIS/ADC/Gatwick(EGKK).json:1]: Sure. What does the `EGKK COM ATIS` mean in it?

**Comment by fcooper94** [comment]
https://github.com/lennycolton/vATIS-Profiles/blob/main/Heathrow.json

Here it is.


Sent from my iPhone

On 18 Apr 2024, at 08:39, Ben Walker ***@***.***> wrote:

﻿

@RedstonePilot commented on this pull request.

________________________________

On UK/vATIS/ADC/Gatwick(EGKK).json<https://github.com/VATSIM-UK/uk-controller-pack/pull/669#discussion_r1570650392>:

Sure. What does the EGKK COM ATIS mean in it?

**Comment by RedstonePilot** [comment]
I'll replace it in a bit :)

File: UK/vATIS/ADC/Gatwick(EGKK).json
Comment (SamLefevre) [reply UK/vATIS/ADC/Gatwick(EGKK).json:1]: Well I know what EGKK and ATIS mean! But please leave it as is, it is based off real world.

Comment (RedstonePilot) [reply UK/vATIS/ADC/Gatwick(EGKK).json:1]: Fair enough 😆 . Wasn't planning on removing it.

Comment (RedstonePilot) [reply UK/vATIS/ADC/Gatwick(EGKK).json:1]: The voice seems to be messed up. Will quickly fix that then push the changes

**Review by SamLefevre — APPROVED** [review]

**Pull request merged**